docker/environment.env

```diff
@@ -0,0 +1,8 @@
+PGADMIN_DEFAULT_EMAIL=admin@admin.pl
```

Comment:
Maybe some COI email?

Reply:
If someone proposes one, I have no idea what to put in as it's just used inside the container and nowhere else